#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Name exception with a message raised."""
    raise NameError(message)
